dist/index.html

```diff
@@ -1,11 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
-<script defer src="main.js"></script></head>
-<body>
-    <h1>Hello webpack!</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To DO List</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"
+    />
+  <script defer src="main.js"></script></head>
+
+  <body>
+    <div class="container">
+      <h1>To Do list</h1>
+      <form class="todo-form">
+        <input type="text" class="todo-input" placeholder="Add to your list..." />
+        <button type="submit" class="add-button">Add</button>
+      </form>
+      <ul class="todo-items">
+        <li class="item" data-key="1594003133171">
+          <input class="checkbox" type="checkbox">
+            Study JS
+          <button class="delete-button">X</button>
+        </li>
+        <li class="item" data-key="1594003133171">
+          <input class="checkbox" type="checkbox">
+            Study CSS
+          <button class="delete-button">X</button>
+        </li>
+        <li class="item" data-key="1594003133171">
+          <input class="checkbox" type="checkbox">
+            Watch football
+          <button class="delete-button">X</button>
+        </li>
+      </ul>
```

Comment:
- [ ] These tasks should be rendered from JS as a part of dynamic content. Please make sure to load JS has an array of some simple to do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. Eg.:

```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 1,
  },
  {
    description: 'An example description in string',
    completed: false,
    index: 2,
  }
];
```

Reply:
Okay, let me implement it